---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: Assessing the functional relevance of splice isoforms
subtitle: ''
summary: ''
authors:
- Fernando Pozo
- Laura Martinez-Gomez
- Thomas A Walsh
- José Manuel Rodriguez
- Tomas Di Domenico
- Federico Abascal
- Jesús Vazquez
- Michael L Tress
tags: []
categories: []
date: '2021-05-01'
lastmod: 2021-05-31T13:13:00+02:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-05-31T11:12:53.149540Z'
publication_types:
- '2'
abstract: Alternative splicing of messenger RNA can generate an array of mature transcripts,
  but it is not clear how many go on to produce functionally relevant protein isoforms.
  There is only limited evidence for alternative proteins in proteomics analyses and
  data from population genetic variation studies indicate that most alternative exons
  are evolving neutrally. Determining which transcripts produce biologically important
  isoforms is key to understanding isoform function and to interpreting the real impact
  of somatic mutations and germline variations. Here we have developed a method, TRIFID,
  to classify the functional importance of splice isoforms. TRIFID was trained on
  isoforms detected in large-scale proteomics analyses and distinguishes these biologically
  important splice isoforms with high confidence. Isoforms predicted as functionally
  important by the algorithm had measurable cross species conservation and significantly
  fewer broken functional domains. Additionally, exons that code for these functionally
  important protein isoforms are under purifying selection, while exons from low scoring
  transcripts largely appear to be evolving neutrally. TRIFID has been developed for
  the human genome, but it could in principle be applied to other well-annotated species.
  We believe that this method will generate valuable insights into the cellular importance
  of alternative splicing.
publication: '*NAR Genomics and Bioinformatics*'
url_pdf: https://doi.org/10.1093/nargab/lqab044
doi: 10.1093/nargab/lqab044
---
