---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: An analysis of tissue-specific alternative splicing at the protein level
subtitle: ''
summary: ''
authors:
- Jose Manuel Rodriguez
- Fernando Pozo
- Tomas di Domenico
- Jesus Vazquez
- Michael L. Tress
tags:
- '"Alternative splicing"'
- '"Proteomic databases"'
- '"Proteomics"'
- '"Cytoskeleton"'
- '"Muscle proteins"'
- '"Muscle tissue"'
- '"Tissue proteins"'
- '"Transcriptome analysis"'
categories: []
date: '2020-10-05'
lastmod: 2020-11-04T20:05:21+01:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-11-04T19:05:21.485528Z'
publication_types:
- '2'
abstract: 'The role of alternative splicing is one of the great unanswered questions
  in cellular biology. There is strong evidence for alternative splicing at the transcript
  level, and transcriptomics experiments show that many splice events are tissue specific.
  It has been suggested that alternative splicing evolved in order to remodel tissue-specific
  protein-protein networks. Here we investigated the evidence for tissue-specific
  splicing among splice isoforms detected in a large-scale proteomics analysis. Although
  the data supporting alternative splicing is limited at the protein level, clear
  patterns emerged among the small numbers of alternative splice events that we could
  detect in the proteomics data. More than a third of these splice events were tissue-specific
  and most were ancient: over 95% of splice events that were tissue-specific in both
  proteomics and RNAseq analyses evolved prior to the ancestors of lobe-finned fish,
  at least 400 million years ago. By way of contrast, three in four alternative exons
  in the human gene set arose in the primate lineage, so our results cannot be extrapolated
  to the whole genome. Tissue-specific alternative protein forms in the proteomics
  analysis were particularly abundant in nervous and muscle tissues and their genes
  had roles related to the cytoskeleton and either the structure of muscle fibres
  or cell-cell connections. Our results suggest that this conserved tissue-specific
  alternative splicing may have played a role in the development of the vertebrate
  brain and heart.'
publication: ''
url_pdf: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008287
doi: 10.1371/journal.pcbi.1008287
---
